lib/localvore_sdk.ex

```diff
@@ -54,6 +54,8 @@ defmodule LocalvoreSdk do
     ] ++ headers
   end
 
+  defp process_response_body(""), do: ""
```

Comment:
feel like we should add a nil check here for good measure. I also sorta feel like returning a nil would be better. @doomspork thoughts?